qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -138,15 +138,23 @@ def write_local_config(self):
         self.test_dev = mon_sock + ':/' + self.test_dirname
         self.scratch_dev = mon_sock + ':/' + self.scratch_dirname
 
+        mntopts_list = self.mount_a.client_config.get('mntopts', [])
+        mntopts = ""
+        if mntopts_list != None:
+            for opt in mntopts_list:
+                mntopts = mntopts + "," + opt
```

Comment:
IMO this could be simplified by using:

`mntopt = ','.joint(mntopts_list)`

Reply:
Ah, yeah.  I keep writing C in Python.  Thanks, I'll fix that.